uwblueprint/focus-on-nature
# Aaron/fix email validation

## Notion ticket link
[Validate email in registration](https://www.notion.so/uwblueprintexecs/Validate-email-in-registration-92b1f52ea9934d6f8f7997ed02ff9d22)


## Implementation description
* Added error handling via toast for invalid email address input when adding/waitlisting campers


## Steps to test
1. Go to http://localhost:3000/register/camp/6396319f01dd5e987a03b516
2. Try to register with an invalid email. This should prompt an error message
3. Try to register with a valid email. This should take you to the next step in registration


## What should reviewers focus on?
* Whether I considered the waitlist flow in addition to the normal registration flow


## Checklist
